ddtrace/ext/ci.py

```diff
@@ -222,7 +222,6 @@ def extract_jenkins(env):
         git.COMMIT_SHA: env.get("GIT_COMMIT"),
         git.REPOSITORY_URL: env.get("GIT_URL"),
         git.TAG: tag,
-        JOB_URL: env.get("JOB_URL"),
```

Comment:
Was this field only being used for jenkins?

Reply:
There is no easy and correct way to generate `JOB_URL` for Jenkins.

Reply:
Ah! I missed that this was under `extract_jenkins` -- silly comment haha